packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -69,80 +77,95 @@ def create_client(
     measurement_service_class: str,
     module_name: Optional[str],
     class_name: Optional[str],
+    all: Optional[bool],
     directory_out: Optional[str],
 ) -> None:
     """Generates a Python Measurement Plug-In Client module for the measurement service.
 
     You can use the generated module to interact with the corresponding measurement service.
 
     MEASUREMENT_SERVICE_CLASS: The service class of the measurement.
```

Comment:
Why is this in caps?

Reply:
While we run `ni-measurement-plugin-client-generator --help`, usage (auto-generated) and our documentatation will be displayed (as in the attached image). 
The argument(in our case, measurement_service_class) , that we provide will be displayed in caps in the usage by default. So to avoid confusion, we have documented it in caps to match it.
Followed this convention from `Plugin-generator`.
![image](https://github.com/user-attachments/assets/1e7cbeab-fd92-440a-a9c8-2c42357578b5)